components/admin_console/database_settings.jsx

```diff
@@ -124,6 +120,12 @@ export default class DatabaseSettings extends AdminSettings {
                             defaultMessage='Driver Name:'
                         />
                     </label>
```

Comment:
Looks like there is a syntax error around here, maybe a dev can point me to what's wrong

Reply:
The `helpText={...}` part needs to be inside of a `*Setting` component. Since this isn't using one of those components, we'll need to add that manually. I'll fix it